mcts.py

```diff
@@ -103,7 +103,9 @@ def child_Q(self):
 
     @property
     def child_U(self):
-        return (FLAGS.c_puct * math.sqrt(1 + self.N) *
+        # NOTE(#379): Expanding self counts as 1, self.N - 1 = sum(child_N).
```

Comment:
I think you can revert this change if you don't increment current.N at the top of the while loop?

Reply:
I disagree,

The paper calls for sqrt(sum(child_n))
* the +1 is not called for
* self.N != sum(child_n), it's sum(child_n) + 1
* the while loop increments N out of order previously resulting in an additional +1 (now handled in that loop)

Reply:
After meditating on it, I think it's hard to argue in favor of either `sqrt(self.N)` or `sqrt(max(1, self.N-1))`. I'd prefer the former for simplicity's sake. AGZ paper doesn't really say one way or another.

Reply:
the paper is pretty clear that the thing in the sqrt is the sum of the visits of the siblings (i.e., parent.N - 1)